app/components/draft_post_header/ProfileAvatar/index.tsx

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {Image} from 'expo-image';
```

Comment:
I feel like this is the same copy as https://github.com/mattermost/mattermost-mobile/blob/mobile-drafts/app/screens/gallery/footer/avatar/index.tsx.  Why not make it a common component?

Reply:
The styles are different, we need to pass the required styles for both the image and view container, should we do it for better have a separate component for it?